src/main/java/httpserver/models/Response.java

```diff
@@ -4,14 +4,16 @@
 
 import java.util.HashMap;
 
+import static httpserver.utils.Constants.PROTOCOL;
+
 public class Response {
     public String statusCode;
     public String body;
     public String protocol;
     public HashMap<String, String> headers;
 
     public Response(String statusCode, HashMap<String, String> headers, String body) {
-        this.protocol = Constants.PROTOCOL;
+        this.protocol = PROTOCOL;
```

Comment:
I often like to import the class and do Constants.PROTOCOL so that when the code reader sees this part of the code, they know that the variable is coming from this other file. PROTOCOL alone could feel like its coming from this file and then one wonders why this variable would be in this file only....
Sometimes its a choice on what reads better AND requires less brain power to understand something. Saving 30 seconds of brain processing power each time adds up to a lot. 
Food for thought :)

Reply:
Good point! Wouldn't have thought of that.